manager/allocator/cnmallocator/networkallocator.go

```diff
@@ -603,9 +604,16 @@ func (na *cnmNetworkAllocator) allocateVIP(vip *api.Endpoint_VirtualIP) error {
 			return err
 		}
 	}
+	if localNet.nw.IPAM != nil && localNet.nw.IPAM.Driver != nil {
```

Comment:
Can I request you to please add comments in the source ? Its probably obvious to those familiar with the code, but not so much for first time readers. Same comment for other changes also.

Reply:
sure